gnupg/_parsers.py

```diff
@@ -1253,6 +1253,8 @@ def _handle_status(self, key, value):
         if key == "IMPORTED":
             # this duplicates info we already see in import_ok & import_problem
             pass
+        elif key == ("PINENTRY_LAUNCHED"):
```

Comment:
I think you might not want those parentheses there? I can fix it after merging.

Reply:
Ah; good catch. I think I probably started with a `key in ("…", "…")` construct. Sorry.